lib/gooddata/lcm/actions/collect_dynamic_schedule_params.rb

```diff
@@ -43,13 +43,15 @@ def call(params)
           input_data = without_check(PARAMS, params) do
             File.open(data_source.realize(params), 'r:UTF-8')
           end
-          GoodData.logger.debug("Input data: #{input_data.read}")
 
           schedule_params = {}
 
           CSV.foreach(input_data, :headers => true, :return_headers => false, encoding: 'utf-8') do |row|
-            GoodData.logger.debug("Processing row: #{row}")
-            results << row.to_hash
+            safe_to_print_row = Hash[row.to_hash.map do |k, v|
```

Comment:
warning: Lint/UnusedBlockArgument: Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used. (rubocop)